src/main/java/org/fundacionjala/sfdc/pageobjects/login/SalesForceLogIn.java

```diff
@@ -46,15 +48,15 @@
      * @param url the page.
      */
     public SalesForceLogIn(final String url) {
-        LOGGER.info("entro");
+      //  LOGGER.info("entro");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)